### ISSUE TITLE
Filter scenes by excluding a tag.

### ISSUE THREAD

**Issue by StagBullSwitch** [stashapp/stash#4537]
**Is your feature request related to a problem? Please describe.**
When trying to filter scenes by tag, you can include tags to search for, but can't exclude them.

**Describe the solution you'd like**
An option to exclude specific tags, rather than just the blanket "Is Null."

As an example, having a mix of pro scenes and amateur camgirl (tagged with _camgirl_) scenes, and wanting to find everything that fits a "blowjob" search query that doesn't have the _camgirl_ tag, so it only lists pro scenes.

**Comment by WithoutPants** [comment]
![image](https://github.com/stashapp/stash/assets/53250216/f35f0b38-9274-4099-9552-f57fa6750124)

Click on the `-` button to exclude tags.

**Comment by StagBullSwitch** [comment]
Ahh, ok.  That was me misunderstanding the interface, then.

Thanks.

**Comment by WithoutPants** [comment]
Yes, this points to a smell with the UX for that component. I'll keep it in mind for improvement.

**Comment by WeedLordVegeta420** [comment]
Looking under "Includes" in order exclude a tag is definitely counter-intuitive, on top of the "-" being relatively easy to overlook.

**Comment by StagBullSwitch** [comment]
It makes sense when you figure out how it works, as the text box is a wildcard/regex entry like the other searches, _or_ you can include and exclude individual tags.

You're right, though....it's counter-intuitive at first.  Not sure what the improvement would be, other than maybe better labelling.

**Comment by ALonelyJuicebox** [comment]
I know this was closed as not planned but this definitely could use a UX improvement-- I use Stash almost daily and I didn't realize this feature was even in here until this past week!

Given that the "plus" sign adds the tag to the includes list, the minus sign invokes a "OK, this will remove the tag I just added to the includes list". It's definitely not clear that it has an entirely different behaviour.

Maybe we change the minus sign to a small "Exclude Tag" button"
Or maybe an additional filter of "Tags (Excluded)" is warranted here to remove confusion.
